docs/core/integration-quality-scale/rules/iqs052.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "IQS052 - The documentation provides examples the user can use. For example (snippets of) configuration, automations, blueprints, scripts, or dashboards"
+---
+
+IQS052 is as follows:
+
+The documentation provides examples the user can use. For example (snippets of) configuration, automations, blueprints, scripts, or dashboards
+
+## Reasoning
+
+If devices are likely going to be used for a few use cases, we can provide blueprints or scripts to help them set it up.
```

Comment:
I think we should say that we want a limited set of examples, ie the most common or useful ones that shows how to use the integration. We don't want the docs to replace the forum or blueprint exchange to collect all sorts of examples.

Keeping the docs succinct is important.